### DISCUSSION TITLE
Setting UP ISR

### DISCUSSION THREAD

**Discussion by webjokerw** [vercel/community#1217] [Help] [answered]
So guys, i'm trying to make ISR work but dosn't manage to get it working. Followed the step's but without success. 
Frist, created the file named revalidate.js
![image](https://user-images.githubusercontent.com/120755052/211058276-9e1a2ef9-9011-4b4d-b473-9875c221e58f.png)

So trying to show that code:
![image](https://user-images.githubusercontent.com/120755052/211060179-aeca0fba-62ca-4a4a-8310-62adf6d43dbb.png)

And when accessing the page 
![image](https://user-images.githubusercontent.com/120755052/211060421-b017952b-c85d-471f-adfc-45a856c1479c.png)

Someone can help-me?

**Comment by amyegan** [comment, marked as answer]
Hi @webjokerw! It looks like you might have a file structure problem. You have `auti.js` set up like a page but it is located in your `api` directory. You will also need to change your `res.revalidate` param to be the relative URL path rather than the file path.

You can learn more about ISR setup in Vercel's [ISR Quickstart](https://vercel.com/docs/concepts/incremental-static-regeneration/quickstart#) and the [Next.js Incremental Static Regeneration](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) documentation.

I also put together a minimal version of what I _think_ you're trying to do, in case it helps to see it in a project context: https://github.com/amyegan/nextjs-isr. Please let me know if I misunderstood, or if you need more help with this!
